### PR TITLE
Sbudhirajadoc tdx readme

### PR DESCRIPTION
**Pull request by sbudhirajadoc**
README Update with new features.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,51 @@
...
+</div>
 
-Oh! And you can also provide general feedback, request product enhancements, start discussions with other Einstein for Developers users or the product team, and share best practices, using the [Einstein for Developers Trailblazer Group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94V000000oRJs?tab=discussion&sort=LAST_MODIFIED_DATE_DESC).
+Einstein for Developers (Beta) is an AI-powered developer tool that’s available as an easy-to-install Visual Studio Code extension built using CodeGen, the secure, custom AI model from Salesforce. Head over to the [Einstein for Developers Documentation](https://developer.salesforce.com/tools/vscode/en/einstein/einstein-overview) to learn more.
 
-We also encourage you to give us feedback on the _generated code_ using 👍, 👎 and comments inside the app. 
+## Get Started
+
+Einstein for Developers is available in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-einstein-gpt) and the [Open VSIX Registry](https://open-vsx.org/extension/salesforce/salesforcedx-einstein-gpt).
```
Comment (AnanyaJha) [inline README.md:11]:
```suggestion
Einstein for Developers is available in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-einstein-gpt) and the [Open VSX Registry](https://open-vsx.org/extension/salesforce/salesforcedx-einstein-gpt).
```

File: images/einstein-inline-create-account.png
Comment (AnanyaJha) [inline images/einstein-inline-create-account.png:1]: where does this image show up? 👁️

File: images/einstein-inline-autocomplete.gif
Comment (AnanyaJha) [inline images/einstein-inline-autocomplete.gif:1]: so cool!

File: README.md [hunk 2]
```diff
@@ -1,23 +1,51 @@
...
+
+* Quickly generate code suggestions using natural language instructions.
+
+* Use the inline code auto-completion feature to get completions for Apex, Javascript, HTML and CSS code.
+
+<div align=center>
+<img src="https://github.com/forcedotcom/Einstein-GPT-for-Developers/blob/main/images/einstein-inline-autocomplete.gif" width="750">
+</div>
+  
+* Generate Apex Unit Tests to quickly get to required code coverage.
```
Comment (AnanyaJha) [inline README.md:31]:
```suggestion

* Generate Apex Unit Tests to quickly get to required code coverage.
```

File: images/einstein-inline-create-account.png
Comment (sbudhirajadoc) [reply images/einstein-inline-create-account.png:1]: that was in the older version of the readme. from last time.

File: README.md [hunk 3]
```diff
@@ -1,23 +1,51 @@
...
+
+## Disable Einstein for Developers
+Einstein for Developers is enabled by default. You can easily disable or uninstall the Einstein for Developers extension at any time. Bring up the Extensions view by clicking on the Extensions icon in the Activity Bar on the side of VS Code or the **View: Show Extensions** command (⇧⌘X). Search for "Einstein for Developers" in installed extensions list, and then select **Disable** or **Uninstall**.
+
+
+## What can Einstein for Developers Do?
+
+Use Einstein for Developers, an AI-powered developer tool, to enhance developer productivity.
+
+* Quickly generate code suggestions using natural language instructions.
```
[the remark this answers is not in the document]
Comment (sbudhirajadoc) [reply README.md:23]: yes working on that gif as we speak.

File: README.md
Comment (kfidelak94) [inline README.md:1]: "This repo will be the primary means of direct communication between our Beta users and the team behind the Einstein for Developers!"

Remove "the" before Einstein for Developers

File: README.md [hunk 4]
```diff
@@ -1,23 +1,54 @@
...
+
+## Disable Einstein for Developers
+Einstein for Developers is enabled by default. You can easily disable or uninstall the Einstein for Developers extension at any time. Bring up the Extensions view by clicking on the Extensions icon in the Activity Bar on the side of VS Code or the **View: Show Extensions** command (⇧⌘X). Search for "Einstein for Developers" in installed extensions list, and then select **Disable** or **Uninstall**.
+
+
+## What can Einstein for Developers Do?
+
+Use Einstein for Developers, an AI-powered developer tool, to enhance developer productivity.
+
+* **Use the inline code auto-completion feature to get completions for Apex, Javascript, HTML and CSS code**
```
Comment (AnanyaJha) [inline README.md:23]: Can we make it clear the JS, HTML, and CSS is for LWC?

File: README.md [hunk 5]
```diff
@@ -1,23 +1,54 @@
...
+
+<div align=center>
+<img src="https://github.com/forcedotcom/Einstein-GPT-for-Developers/blob/main/images/einstein-inline-autocomplete.gif" width="750">
+</div>
+  
+* **Generate Apex Unit Tests to quickly get to required code coverage**
```
Comment (AnanyaJha) [inline README.md:29]: Can we update the "required code coverage" with the language we agreed on from the walkthrough this morning?

File: README.md (hunk 4 above)
Comment (kfidelak94) [reply README.md:23]: +1

File: README.md [hunk 6]
```diff
@@ -1,23 +1,54 @@
...
+
+Use Einstein for Developers, an AI-powered developer tool, to enhance developer productivity.
+
+* **Use the inline code auto-completion feature to get completions for Apex and LWC (JS, HTML and CSS) code**
+
+<div align=center>
+<img src="https://github.com/forcedotcom/Einstein-GPT-for-Developers/blob/main/images/einstein-inline-autocomplete.gif" width="750">
+</div>
+  
+* **Generate unit tests for your code to help improve your code coverage**
```
Comment (AnanyaJha) [inline README.md:29]:
```suggestion
* **Generate Apex unit tests for your code to help improve your code coverage**
```

File: README.md [hunk 7]
```diff
@@ -1,23 +1,54 @@
...
+
+<div align=center>
+<img src="https://github.com/forcedotcom/Einstein-GPT-for-Developers/blob/main/images/einstein-tcg.gif" width="750">
+</div>
+
+* **Quickly generate code suggestions using natural language instructions**
```
Comment (AnanyaJha) [inline README.md:35]:
```suggestion
* **Quickly generate Apex code suggestions using natural language instructions**
```

**Review by AnanyaJha — APPROVED** [review]
Yay!! Thanks for bearing with my edits :)

**Pull request merged**